bridge/client/app/_views/ktb-common-use-cases-view/ktb-markdown/ktb-markdown.component.scss

```diff
@@ -0,0 +1,2 @@
+@import 'client/variables';
+@import 'node_modules/@dynatrace/barista-components/core/src/style/variables';
```

Comment:
You can just delete the file because there isn't something in it. Also make sure to adjust the component and remove the styleURL.